packages/core/js-client/src/compilerSupport/registerService.ts

```diff
@@ -14,66 +14,63 @@
  * limitations under the License.
  */
 
-import type { ServiceDef, ServiceImpl } from "@fluencelabs/interfaces";
-
 import { FluencePeer } from "../jsPeer/FluencePeer.js";
 import { logger } from "../util/logger.js";
 
 import { registerGlobalService, userHandlerService } from "./services.js";
+import { ServiceImpl } from "./types.js";
 
 const log = logger("aqua");
 
 interface RegisterServiceArgs {
   peer: FluencePeer;
-  def: ServiceDef;
   serviceId: string | undefined;
   service: ServiceImpl;
 }
 
+const findAllPossibleServiceMethods = (service: ServiceImpl): Set<string> => {
```

Comment:
doc

Reply:
what does `ServiceMethod` mean?

Reply:
names it
`findAllPossibleRegisteredServiceFunctions`